scss/_button-group.scss

```diff
@@ -119,17 +113,14 @@
   align-items: flex-start;
   justify-content: center;
 
-  .btn,
-  .btn-group {
+  > .btn,
+  > .btn-group {
     width: 100%;
   }
 
-  > .btn + .btn,
-  > .btn + .btn-group,
-  > .btn-group + .btn,
-  > .btn-group + .btn-group {
+  > .btn:not(:first-child),
+  > .btn-group:not(:first-child), {
```

Comment:
This doesn't look ok

Reply:
My all test case is here. There is no visual difference 👍 
* Before: https://codepen.io/anon/pen/RqbRRW
* After: https://codepen.io/anon/pen/QJLNPv

Reply:
Is there a linter for stuff like that in the future?